app/database/queries/breeze_buddy/analytics.py

```
@@ -845,3 +849,65 @@ def get_analytics_lead_status_counts_total_query(
+
+    if group_by:
+        # Grouped inbound count - returns one row per group
+        text = f"""
+            SELECT 
+                lct.{group_by},
+                COUNT(*) as inbound_calls
+            FROM "{LEAD_CALL_TRACKER_TABLE}" lct
+            {where_clause}
+            GROUP BY lct.{group_by}
+            ORDER BY inbound_calls DESC;
+        """
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)